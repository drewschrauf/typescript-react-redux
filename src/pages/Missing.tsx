import React from 'react';

const Missing = () => <h1>There&apos;s nothing here</h1>;
export default Missing;
