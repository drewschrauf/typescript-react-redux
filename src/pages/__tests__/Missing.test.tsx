import React from 'react';
import { mount } from 'enzyme';

import Missing from '../Missing';

describe('Missing', () => {
  it('should match snapshot', () => {
    const root = mount(<Missing />);
    expect(root.html()).toMatchSnapshot();
  });
});
