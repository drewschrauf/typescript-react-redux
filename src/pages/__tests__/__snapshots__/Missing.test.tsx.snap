// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Missing should match snapshot 1`] = `"<h1>There's nothing here</h1>"`;
